apps/web/src/pages/templates/TemplatesDigestPlaygroundPage.tsx

```diff
@@ -43,7 +43,7 @@ export const TemplatesDigestPlaygroundPage = () => {
         <DigestWorkflowHolder />
       </Stack>
       <Stack mt={40} mb={40} align="center">
-        <Button onClick={() => navigate(parseUrl(ROUTES.TEMPLATES_EDIT_TEMPLATEID, { templateId }))}>
+        <Button onClick={() => navigate(`${parseUrl(ROUTES.TEMPLATES_EDIT_TEMPLATEID, { templateId })}?tour=digest`)}>
```

Comment:
the query param that says to show the digest tour with hints

Reply:
I like it but I have some questions regarding the UX here:
1. in a scenario where the user:  entered the tour => and goes back, he will miss the tour because not he won't have the proper URL.
2. page refresh, no matter if the user skipped the tour or not he will get the tour again at the current state.

My other thought about it is if we will add more tours in the future, for example "user added filter step for the first time" and we would want to execute a tour, I guess my question should we persist this data on DB? (most definitely not related to this cycle 🫣)

Reply:
@djabarovgeorge 
1. I'm not sure if I understand what you mean, but if the user navigates from the Templates page to the Digest Playground page and then back, he will miss the tour.
2. Yes, we wanted to have it like it is, whenever the user enters the Workflow Editor from the Digest Playground we want to show the tour. 
3. Usually it's enough to store this information in the local storage.

Let's see what Nikolay will say after he plays with it.

Reply:
Local storage sound good to me :)